pytorch_pfn_extras/writing.py

```diff
@@ -500,3 +500,21 @@ def create_queue(self):
 
     def create_consumer(self, q):
         return multiprocessing.Process(target=self.consume, args=(q,))
+
+
+class TensorBoardWriter(object):
+    def __init__(self, savefun=None, fs=None, out_dir=None, **kwds):
+        import torch.utils.tensorboard
+        self._writer = torch.utils.tensorboard.SummaryWriter()
```

Comment:
`torch.utils.tensorboard.SummaryWriter()` is initialized with the default arguments. So the log file is saved in `runs/CURRENT_DATETIME_HOSTNAME` as described in the document.
https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter

How about use `out_dir` to initialize it?